os.md

```diff
@@ -31,3 +31,9 @@ Ubuntu 22.04 | All supported kernels
 Ubuntu 20.04 | >= 5.4.0-1047-aws, >= 5.8.0-1034-aws, >= 5.11.0-1009-aws
 Ubuntu 18.04 | >= 4.15.0-1101-aws, >= 5.4.0-1047-aws
 Redhat Entreprise Linux 8 | >= 4.18.0-305.10
+
+# Operating Systems not supported by Graviton based instances
```

Comment:
I agree with Arthur. The text should say "Operating systems which do not support Graviton".

Reply:
done